README.md

```diff
@@ -15,39 +15,72 @@
   <img src="assets/res.webp"/>
 </p>
 
-# About
-
 This GTK theme is based on the [Colloid](https://github.com/vinceliuice/Colloid-gtk-theme) theme made by [Vinceliuice](https://github.com/vinceliuice)
 
-## Usage
+# Usage
 
 ### Requirements
 
-- GTK `>=3.20`
+- GTK >=3.20
+- `python3`
 - `gnome-themes-extra` (or `gnome-themes-standard`)
 
-### Installation
+If you want to build the theme, make sure these packages are installed:
+- `sassc`
+- `inkscape`
+- `optipng`
+
+In order to make sure you're running Python 3, run ``python --version`` and it should output ``Python 3.x.x``
 
-1. Download and extract the theme zip from [releases](https://github.com/catppuccin/gtk/releases/).
-2. Move the theme folder to **".themes"** in your home directory. **(~/.themes)** (Skip this step if you are using the AUR package)
-3. Select the downloaded theme via your desktop specific tweaks application (**gnome-tweaks** on Gnome 3+).
+### Installing the theme manually
 
-### For Arch Linux users
+1. Download and extract the theme zip from the [releases](https://github.com/catppuccin/gtk/releases/) page.
+2. Move the theme folder to the **~/.local/share/themes** directory (Skip this step if you're using the AUR package)
+3. Select the downloaded theme via your desktop specific tweaks application (GNOME Tweaks on GNOME 3+)
+4. To theme other apps that are using GTK, make sure to run the following command:
+```bash
+$ mkdir -p "${HOME}/.config/gtk-4.0" && ln -sf "${THEME_DIR}/gtk-4.0/assets" "${HOME}/.config/gtk-4.0/assets" && ln -sf "${THEME_DIR}/gtk-4.0/gtk.css" "${HOME}/.config/gtk-4.0/gtk.css" && ln -sf "${THEME_DIR}/gtk-4.0/gtk-dark.css" "${HOME}/.config/gtk-4.0/gtk-dark.css"
```

Comment:
I think this would be better for readablity if it was over multiple lines and without the $

Reply:
oh yeah i forgot about that

Reply:
ima change it

Reply:
We also should include the `export` for `THEME_DIR` here, so users can more easily copy paste the commands.
```suggestion
export THEME_DIR="~/.local/share/themes/catppuccin-mocha-mauve-standard+default"
mkdir -p "${HOME}/.config/gtk-4.0" && 
ln -sf "${THEME_DIR}/gtk-4.0/assets" "${HOME}/.config/gtk-4.0/assets" &&
ln -sf "${THEME_DIR}/gtk-4.0/gtk.css" "${HOME}/.config/gtk-4.0/gtk.css" &&
ln -sf "${THEME_DIR}/gtk-4.0/gtk-dark.css" "${HOME}/.config/gtk-4.0/gtk-dark.css"
```